content/articles/data-transformation-with-linq/index.md

```diff
@@ -0,0 +1,189 @@
+
+### Introduction
+LINQ (Language-Integrated Query) is used for more than just data retrieval. It's also a useful tool for data transformation. New output may be gotten using LINQ by capturing source sequences and converting them in various ways.
+
+Grouping and sorting are techniques that usually change the sequence of a query, but the same change can be achieved without using those techniques. This is achievable when a learner or a programmer uses the `select` clause. For example, a programmer may perform or carry out some of the following tasks.
+1. In creating one output query consisting of another type, a programmer can combine several input queries.
+2. A programmer may still opt to take a few elements from each element in the source sequence to create an output sequence.
+3. By putting into consideration the results of operations on source data and several other things, an output sequence can still be obtained.

```

Comment:
An introduction is meant to introduce the reader to your article. Be precise and go straight to the point, a paragraph is just enough, any other content move to the body.